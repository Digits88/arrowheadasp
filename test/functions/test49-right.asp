<!-- Test Right function.
-->

Right("abcdef", 1) = <%= Right("abcdef", 1) %><br>
Right("abcdef", 0) = <%= Right("abcdef", 0) %><br>
Right("abcdef", 8) = <%= Right("abcdef", 8) %><br>
IsNull(Right(Null, 2)) = <%= IsNull(Right(Null, 2)) %><br>
