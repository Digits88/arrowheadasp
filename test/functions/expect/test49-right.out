<!-- Test Right function.
-->

Right("abcdef", 1) = f<br>
Right("abcdef", 0) = <br>
Right("abcdef", 8) = abcdef<br>
IsNull(Right(Null, 2)) = True<br>
